src/lib/modernExtend.ts

```diff
@@ -690,11 +697,24 @@ export function deviceEndpoints(args: {endpoints: {[n: string]: number}, multiEn
     return result;
 }
 
-export function ota(args: {definition: DefinitionOta}): ModernExtend {
-    return {
-        ota: args.definition,
+export interface OtaArgs {
+    definition?: DefinitionOta, bindOtaCluster?: boolean, otaClusterId?: number,
+}
+export function ota(args?: OtaArgs): ModernExtend {
+    let configure: Configure;
+    if (args.bindOtaCluster === true && args.otaClusterId) {
+        configure = async (device, coordinatorEndpoint, logger) => {
+            const endpoint = device.getEndpoint(args.otaClusterId);
+            await reporting.bind(endpoint, coordinatorEndpoint, ['genOta']);
```

Comment:
I don't expect binding `genOta` is really needed

Reply:
I asked the lead firmware developer for these sensors and genOta bind is indeed useless. I'll remove it